canopen/network.py

```diff
@@ -247,7 +247,7 @@ class PeriodicMessageTask(object):
     CyclicSendTask
     """
 
-    def __init__(self, can_id, data, period, bus):
+    def __init__(self, can_id, data, period, bus, extended=False, remote=False):
```

Comment:
Do you need to use extended frames? I wasn't sure how many actually used that so I have assumed standard frames until someone needs extended. If it is needed than there are a lot of additional changes needed so it should probably done in a new pull request.

Reply:
Ok, I have removed it for now. Like you said if a need extended a will make the changes and we go from their.